src/test/duke/DeadlineCommandTest.java

```diff
@@ -0,0 +1,51 @@
+import duke.command.DeadlineCommand;
+import duke.data.TaskList;
+import duke.data.task.Deadline;
+import duke.storage.Storage;
+import duke.ui.Ui;
+import org.junit.jupiter.api.*;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineCommandTest {
+    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
```

Comment:
As mentioned earlier, final variables/constants should be in full capital letters.